src/kernel-install/meson.build

```diff
@@ -40,9 +40,11 @@ if want_kernel_install
 
         install_data('install.conf',
                      install_dir : kerneldir)
- 
-        install_data('uki.conf',
-                     install_dir : kerneldir)
+
+        if want_ukify
+                install_data('uki.conf',
+                        install_dir : kerneldir)
```

Comment:
Not a big matter, but spurious indentation.

Reply:
You are right! Changes done :)